source/material_model/drucker_prager.cc

```diff
@@ -70,7 +70,7 @@ namespace aspect
               // The negative of the second principle invariant is equal to 0.5 e_dot_dev_ij e_dot_dev_ji,
               // where e_dot_dev is the deviatoric strain rate tensor. The square root of this quantity
               // gives the common definition of effective strain rate.
-              const double edot_ii_strict = (&(this->get_simulator()) == NULL
+              const double edot_ii_strict = (this->simulator_is_initialized()
```

Comment:
Yes. When I try "this->simulator_is_initialized() == NULL", the compiler gives the warning:
warning: comparison between NULL and non-pointer ('bool' and NULL) [-Wnull-arithmetic]

"this->simulator_is_initialized() == false" works. It seems that "NULL" in "simulator_access" is equivalent to bool condition "false" in the "drucker_prager".  What's the reason of this? I'm a little bit confused. @bangerth